app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,117 @@
+<div class="ui buttons">
+  <button class="ui button">All</button>
+  <button class="ui button">Pending</button>
+  <button class="ui button">Accepted</button>
+  <button class="ui button">Confirmed</button>
+  <button class="ui button">Rejected</button>
+</div>
+<button class="ui blue button">Create Session</button>
+<button class="ui blue button">Export as CSV</button>
+<div class="search-box">
+  <div class="ui icon input">
+    <input class="prompt" type="text" placeholder="Search">
+    <i class="search icon"></i>
+  </div>
+  <div class="results"></div>
+</div>
+<br>
+<br>
+<div class="ui grid">
+  <div class="one wide column"> Show </div>
```

Comment:
Remove this dropdown, and search box. As I said, not needed will be handled by #80